.github/workflows/release-bin.yaml

```diff
@@ -73,6 +102,22 @@ jobs:
 
       - name: Compile Native
         run: cargo build --release --locked --manifest-path=roles/jd-client/Cargo.toml
+      
+      - name: Compile Binaries for aarch64-unknown-linux-gnu
+        if: matrix.os == 'ubuntu-20.04'
+        run: cross build --release --locked --manifest-path=roles/jd-client/Cargo.toml --target aarch64-unknown-linux-gnu
+  
+      - name: Compile Binaries for arm-unknown-linux-gnueabi
+        if: matrix.os == 'ubuntu-20.04'
+        run: cross build --release --locked --manifest-path=roles/jd-client/Cargo.toml --target arm-unknown-linux-gnueabi
+
+      - name: Compile Binaries for aarch64-unknown-linux-gnu
+        if: matrix.os == 'ubuntu-20.04'
+        run: cross build --release --locked --manifest-path=roles/jd-client/Cargo.toml --target aarch64-unknown-linux-gnu
+  
+      - name: Compile Binaries for arm-unknown-linux-gnueabi
+        if: matrix.os == 'ubuntu-20.04'
+        run: cross build --release --locked --manifest-path=roles/jd-client/Cargo.toml --target arm-unknown-linux-gnueabi
```

Comment:
this seems duplicated?

Reply:
> this seems duplicated?

I've cleaned them up,  thanks for the review.